### PR TITLE
Update dev-time IIS support in VS topic

### LINKED ISSUE #11868: Update dev-time IIS support in VS topic
I found that the instructions did not work when url was http (vs.net 2019 said that it was unable to start the application pool, even if i manually started the application pool, vs.net 2019 when trying to debug F5 it would stop it..then vs.net would just hang.

After I made it https it all worked.  Is this by design?

### PR DESCRIPTION
**Pull request by guardrex**
Fixes #11868 

* Update, including for 3.0.
* Support for either HTTPS (original coverage) or HTTP (new coverage).
* [Rage Against the Pics](https://www.ratm.com/) ... they're *kill'in the name of!*

### CODE REVIEW

**Comment by shirhatti** [comment]
It's been a while since I've looked at this feature, but I was under the impression that instead of creating a sub-app, we now create a new ApplicationPool per project.

**Comment by guardrex** [comment]
I can't remark on the under-the-covers bits beyond it working as a sub-app (and that setup is managed by the tooling) ... it still works today (VS2019). My main add was to address the reader's ask about using HTTP, and it ✨ *Just Works*:tm: ✨ . However, it won't do both HTTP and HTTPS bindings tho AFAICT. Some part of the built-in tooling support blows up the bindings when both are attempted.

Do you want to ping VS cats?

**Comment by shirhatti** [comment]
I'm okay not addressing that in this PR. As long as we file an issue to follow-up eventually

**Comment by shirhatti** [comment]
I'm guessing we might do a scrub of the docs at the some point for differences between Dev16 and Dev15.

**Comment by guardrex** [comment]
Yeah ... I'm definitely allowing the "it works in VS2019" outcome be my guide here. I imagine (*guess*) that if we ask the maintainer that they're going to say that they didn't change anything.

File: aspnetcore/host-and-deploy/iis/development-time-iis-support.md [hunk 1]
```diff
@@ -1,102 +1,108 @@
...
-
-![Add Website window in IIS showing the binding set for localhost on port 443 with a certificate assigned.](development-time-iis-support/_static/add-website-window.png)
-
-If the IIS installation already has a **Default Web Site** with a host name that matches the app's launch profile URL host name:
-
-* Add a port binding for port 443 (HTTPS).
-* Assign a valid certificate to the website.
+* **Host name** &ndash; Typically, the **Default Web Site** is used with a **Host name** of `localhost`. However, any valid IIS website with a unique host name works.
+* **Site Binding**
+  * For apps that require HTTPS, create a binding to port 443 with a certificate. Typically, the **IIS Express Development Certificate** is used, but any valid certificate works.
```
Comment (Rick-Anderson) [inline aspnetcore/host-and-deploy/iis/development-time-iis-support.md]: > create a binding to port 443 with a certificate. 
How do you create a binding port with a cert? 
I like previous better
The port is set to "443" (HTTPS).
Do you really want to add **binding**? To
For HTTP, use port 80 or create port 80. Binding is probably the correct way to describe this, but I haven't seen it used very often.

File: aspnetcore/host-and-deploy/iis/development-time-iis-support.md [hunk 2]
```diff
@@ -1,102 +1,108 @@
...
+::: moniker range=">= aspnetcore-3.0"
 
-### IIS launch profile
+1. Right-click the project in **Solution Explorer**. Select **Properties**. Open the **Debug** tab.
+1. For **Profile**, select the **New** button. Name the profile "IIS" in the popup window. Select **OK** to create the profile.
+1. For the **Launch** setting, select **IIS** from the list.
+1. Select the check box for **Launch browser** and provide the endpoint URL.
 
-Create a new launch profile to add development-time IIS support:
+   When the app requires HTTPS, use an endpoint with the HTTPS protocol (`https://`). When the app uses HTTP, use an endpoint with the HTTP protocol (`http://`).
```
Comment (Rick-Anderson) [inline aspnetcore/host-and-deploy/iis/development-time-iis-support.md]: When the app requires HTTPS, use an HTTPS endpoint (`https://`). For HTTP, use an HTTP (`http://`) endpoint .

File: aspnetcore/host-and-deploy/iis/development-time-iis-support.md [hunk 3]
```diff
@@ -1,102 +1,108 @@
...
+
+1. Right-click the project in **Solution Explorer**. Select **Properties**. Open the **Debug** tab.
 1. For **Profile**, select the **New** button. Name the profile "IIS" in the popup window. Select **OK** to create the profile.
 1. For the **Launch** setting, select **IIS** from the list.
-1. Select the check box for **Launch browser** and provide the endpoint URL. Use the HTTPS protocol. This example uses `https://localhost/WebApplication1`.
-1. In the **Environment variables** section, select the **Add** button. Provide an environment variable with a key of `ASPNETCORE_ENVIRONMENT` and a value of `Development`.
-1. In the **Web Server Settings** area, set the **App URL**. This example uses `https://localhost/WebApplication1`.
+1. Select the check box for **Launch browser** and provide the endpoint URL.
+
+   When the app requires HTTPS, use an endpoint with the HTTPS protocol (`https://`). When the app uses HTTP, use an endpoint with the HTTP protocol (`http://`).
```
Comment (Rick-Anderson) [inline aspnetcore/host-and-deploy/iis/development-time-iis-support.md]: same

File: aspnetcore/host-and-deploy/iis/development-time-iis-support.md (hunk 1 above)
Comment (guardrex) [reply aspnetcore/host-and-deploy/iis/development-time-iis-support.md]: Can't cross-link this to IIS docs. They're ... mmm ... their docs are "*very interesting*". :smile:

I'll touch it on the next commit given that we're saying to use the Default Web Site; but truly, this is a *can of worms*. The reader has to both play with IIS and look at the IIS topics. We can't cover all of the angles here. I'll add the IIS docs link to the *Additional resources*.

**Comment by shirhatti** [comment]
> Can't cross-link this to IIS docs. They're ... mmm ... their docs are "very interesting". 😄

I'm pretty sure you can xref to IIS. You just need to update your `xref_query_tags` in the `.openpublishing.publish.config.json` file

**Comment by guardrex** [comment]
:smile: That isn't what I meant. Lol

**Pull request merged**